src/views/locale/LocaleDetails.tsx

```diff
@@ -73,13 +74,21 @@ const LocaleDefinitionSection: React.FC<{ locale: LocaleData }> = ({ locale }) =
           )}
         </DetailsField>
       )}
-      {variantTagCode && (
+      {variantTagCodes && variantTagCodes.length > 0 && (
         <DetailsField title="Variant Tag:">
-          {variantTag ? (
-            <HoverableObjectName object={variantTag} />
+          {variantTags && variantTags.length > 0 ? (
+            <span>
+              {}
+              {variantTags.map((tag, idx) => (
+                <React.Fragment key={tag.ID}>
+                  {idx > 0 && ', '}
+                  <HoverableObjectName object={tag} />
+                </React.Fragment>
+              ))}
+            </span>
```

Comment:
We already have a component for this
```suggestion
            <CommaSeparated>
              {variantTags.map((tag, i) => <HoverableObjectName object={tag} key={i} />)}
            </CommaSeparated>
```